src/components/form/Select/OptionItem/MobileOptionItem.tsx

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from 'react';
+import { Listbox } from '@headlessui/react';
+import { CSSType } from 'stitches.config';
+import SelectRadioItem from '@components/filter/MultiSelect/SelectRadioItem';
+
+export interface Option {
+  label: string;
+  // NOTE: null 은 placeholder
+  value: string | null;
+}
+
+interface OptionItemProps {
+  css?: CSSType;
+  option: Option;
+  stringifiedSelectedValue: string;
+}
+
+function MobileOptionItem({ css, option, stringifiedSelectedValue }: OptionItemProps) {
+  const stringifiedValue = useMemo(() => JSON.stringify(option), [option]);
+
+  return (
+    <Listbox.Option key={option.label} value={stringifiedValue}>
+      <SelectRadioItem {...css} value={option.value || ''} isChecked={stringifiedSelectedValue === stringifiedValue} />
```

Comment:
```suggestion
      <SelectRadioItem {...css} value={option.label || ''} isChecked={stringifiedSelectedValue === stringifiedValue} />
```

여기서 `value` 대신 `label`을 써야 할 것 같아!

|option.value 사용 시|option.label 사용 시|
|:-:|:-:|
|<img src="https://user-images.githubusercontent.com/58380158/225358509-f1ba85a7-b1ac-45de-bb1a-07ad086f70c1.png" />|<img src="https://user-images.githubusercontent.com/58380158/225358730-8943fa19-15d8-43b7-ac7f-4d6fefed9eaf.png" />|

Reply:
음~ 이부분은 label을 받는 부분을 추가할게 ! 
form의 select 기준으로 제작하다보니까 label과 value가 같아서 놓친 부분인데,
value는 그대로 두고, 보여지는 부분에서 label을 사용하도록 수정했어!